react/tic-tac-toe/src/redux/match/actions.js

```diff
@@ -21,7 +26,15 @@ const actionsCreators = {
   setLoading: loading => ({
     type: actions.SET_LOADING,
     payload: loading
-  })
+  }),
+  getMatches: () => async dispatch => {
+    const response = await getMatches();
+    if (response.ok) {
+      dispatch({ type: actions.GET_MATCHES_SUCCESS, payload: response, errorMatches: false });
```

Comment:
what is the `errorsMatches` field for? 
Also, in the reducer it's defined as `matchesError`. 
Since you're not using it in your component, nor handling in the redux files, I'd delete it directly